winit-wayland/src/lib.rs

```diff
@@ -120,6 +121,12 @@ impl WindowAttributesWayland {
         self.activation_token = Some(token);
         self
     }
+
+    #[inline]
+    pub fn with_prefer_csd(mut self, prefer_csd: bool) -> Self {
+        self.prefer_csd = prefer_csd;
+        self
+    }
```

Comment:
Should add docs for this one, since it's not clear what it does just from the name.